packages/api/README.md

```diff
@@ -4,8 +4,8 @@ The Polkadot-JS API provides easy-to-use wrappers around JSONRPC calls that flow
 
 The API wrappers provide a standard interface for use -
 
-- A static `.create(<optional WsProvider>, <optional CustomTypes>)` that returns an API instance when connected, decorated and ready-to use.
-- The above is just a wrapper for `new Api(<optional WsProvider>) `, exposing the `isReady` getter
+- A static `.create(<optional ApiOptions>)` that returns an API instance when connected, decorated and ready-to use. ApiOptions can include an optional WsProvider and optional custom type definitions `{ wsProvider: <Optional WsProvider>, types: <Optional CustomTypes> }`.
```

Comment:
{ provider: WsProvider, ... }` as per https://github.com/polkadot-js/api/blob/master/packages/api/src/types.ts#L98 (also has the signer now, we can ignore `source` since it is basically "internal", only used in cloning)

Reply:
I fixed the `wsProvider` to `provider`